include/deal.II/matrix_free/matrix_free.h

```diff
@@ -2471,6 +2516,81 @@ MatrixFree<dim, Number, VectorizedArrayType>::at_irregular_cell(
 
 
 
+template <int dim, typename Number, typename VectorizedArrayType>
+unsigned int
+MatrixFree<dim, Number, VectorizedArrayType>::n_active_fe_indices() const
+{
+  return shape_info.size(2);
+}
+
+
+template <int dim, typename Number, typename VectorizedArrayType>
+unsigned int
+MatrixFree<dim, Number, VectorizedArrayType>::get_cell_active_fe_index(
+  const std::pair<unsigned int, unsigned int> range) const
+{
+  const auto &fe_indices = dof_info[0].cell_active_fe_index;
+
+  if (fe_indices.empty() == true)
+    return 0;
+
+  const auto index = fe_indices[range.first];
+
+#  ifdef DEBUG
+  for (unsigned int i = range.first; i < range.second; ++i)
+    AssertDimension(index, fe_indices[i]);
+#  endif
```

Comment:
I don't think you need the #ifdef here because `AssertDimension` already expands to nothing for non-debug build. There are a few more cases further down.

Reply:
I have removed it!